.github/workflows/ci.yml

```diff
@@ -7,13 +7,13 @@ on:
 jobs:
   sanity:
     name: sanity
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-go@v2
+    - uses: actions/setup-go@v3
       with:
-        go-version: 1.17
+        go-version: 1.19
       id: go
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
Can we set up the caching like in preflight while we're at it?

Reply:
I added the caching.